bpf/process/pfilter.h

```diff
@@ -313,23 +313,39 @@ selector_match(__u32 *f, struct selector_filter *sel,
 				     struct execve_map_value *, struct msg_ns *,
 				     struct msg_capabilities *))
 {
-	int res1 = 0, res2 = 0, res3 = 0, res4 = 0;
+	int res[MAX_SELECTOR_VALUES] = { 0 };
 	__u32 index = sel->index;
 	__u64 len = sel->len;
 	__u64 ty = sel->ty;
+	__u64 i;
+
+	if (len > MAX_SELECTOR_VALUES)
+		len = MAX_SELECTOR_VALUES;
```

Comment:
Maybe better const name for this and also the one in userspace namespace selector? the selectors did grow and it will make it easy for new devs to get it...